lib/jsdom/browser/Window.js

```diff
@@ -34,8 +34,17 @@ const jsGlobals = require("./js-globals.json");
 const GlobalEventHandlersImpl = require("../living/nodes/GlobalEventHandlers-impl").implementation;
 const WindowEventHandlersImpl = require("../living/nodes/WindowEventHandlers-impl").implementation;
 
-exports.createWindow = function (options) {
-  return new Window(options);
+let beforeParse;
+const beforeParseDefaultOptions = { isMain: false };
```

Comment:
A module-level `beforeParseDefaultOptions` is not necessary. Just update `createWindow` to:

```js
exports.createWindow = function (options, beforeParseOptions = { isMain: false }) {
```